src/test/resources/fixtures/hmh/hmh-rce/development/caliperEventNavigationEvent.json

```diff
@@ -0,0 +1,75 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:c942788b-aae8-4a15-8135-b5ec731a0fc3",
+  "type": "NavigationEvent",
+  "actor": {
+    "id": "urn:uuid:ff7c53fd-9024-4d2d-8503-aa3da2cbc73e",
+    "type": "Person"
+  },
+  "action": "NavigatedTo",
+  "object": {
+    "id": "http://rce.localhost/content/book/OPS/page0001.xhtml",
+    "type": "Frame",
+    "name": "Cover Page",
+    "dateCreated": "2019-06-05T14:17:34.739Z",
+    "extensions": {
+      "contentRootId": "9780358087762",
+      "contentRootType": "pubId",
+      "contentVersionId": 0,
+      "userRole": "student",
+      "contentItemId": "xhtml_page0001"
+    },
+    "isPartOf": {
+      "id": "http://rce.localhost/digitalresource/9780358087762",
+      "type": "DigitalResource",
+      "name": "Hero Academy 2019",
+      "isPartOf": {
+        "id": "NoProgrammeId",
+        "type": "DigitalResource",
+        "name": "NoProgramme"
+      },
+      "version": "1.0"
+    },
+    "index": 1
+  },
+  "eventTime": 1559744254740,
+  "referrer": {
+    "id": "http://rce.localhost/content/book/OPS/page0001.xhtml",
+    "type": "WebPage",
+    "name": "Cover Page"
+  },
+  "edApp": {
+    "id": "http://rce.localhost/RCE",
+    "type": "SoftwareApplication",
+    "name": "RCE",
+    "extensions": { "offline": false }
+  },
+  "membership": {
+    "id": "http://rce.localhost/user/ff7c53fd-9024-4d2d-8503-aa3da2cbc73e/class/4444444",
+    "type": "Membership",
+    "organization": {
+      "id": "urn:uuid:48469971-7fa6-44d7-9847-5abf3f684113",
+      "type": "Organization",
+      "subOrganizationOf": {
+        "id": "urn:uuid:80b8ccf8-2996-4669-8dae-2772053d5176",
+        "type": "Organization"
+      }
+    },
+    "roles": ["Learner"],
+    "status": "Active"
+  },
+  "event": {
```

Comment:
I don't think `event` is a valid field - see here https://www.imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html#navigationEvent for valid fields for this NavigationEvent

Reply:
👍

Reply:
What is here in event seems to duplicate what is added under object also, so presumably it can just be deleted.

Reply:
On the edApp I wonder if we need an explicit offline property, I would presume that is could be inferred from a difference in the sensor sendTime (now) and the eventTime (potentially ages ago). Also for time properties the value MUST be expressed using the format YYYY-MM-DDTHH:mm:ss.SSSZ set to UTC with no offset specified.

Reply:
As a general change we should move to the same model for group and membership as now used in the GradedEvent (for 1.1)

Reply:
On both of these events the session object is missing